documentation-site/pages/components/layout-grid.mdx

```diff
@@ -172,6 +173,34 @@ Note how these are defined in a similar fashion to responsive props. Each value
 
 If you want to customize the breakpoints of the grid, you can do so via the `breakpoints` and `mediaQuery` properties. The grid will always pull breakpoints from the theme and correlate responsive props with those breakpoints.
 
+<Example title="Custom css units" path="layout-grid/custom-css-unit.js">
+  <CustomCssUnitExample />
+</Example>
+
+Layout grid uses `px` as a default css unit, you can change that by modifing theme in following manner:
+
+```js
+const theme = {
+  grid: {
+    unit: 'em',
+  },
+};
+```
```

Comment:
Can remove if we swap the example to show theme customizations.

Reply:
If I understood this correctly, this should be part of theme customisation section. right?

Reply:
I have made changes accordingly, Please let me know if I understood this wrong :/